packages/components/modal/src/ModalHeader/ModalHeader.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { cx } from 'emotion';
+import { Close } from '@contentful/f36-icons';
+import { Flex } from '@contentful/f36-core';
+import type { PrimitiveProps } from '@contentful/f36-core';
+import { Button } from '@contentful/f36-button';
+import { Subheading } from '@contentful/f36-typography';
+
+import { getModalHeaderStyles } from './ModalHeader.styles';
+
+export interface ModalHeaderProps extends PrimitiveProps<'div'> {
+  title: string;
+  onClose?: Function;
+  as?: 'div';
+}
+
+export function ModalHeader({
+  onClose,
+  title,
+  testId = 'cf-ui-modal-header',
+  className,
+  ...otherProps
+}: ModalHeaderProps): React.ReactElement {
+  const styles = getModalHeaderStyles();
+
+  return (
+    <Flex
+      {...otherProps}
+      className={cx(styles.root, className)}
+      testId={testId}
+      alignItems="center"
+      justifyContent="space-between"
+    >
+      <Subheading as="h1" isTruncated marginBottom="none">
```

Comment:
Is this correct?

What happens if the page behind the modal has an h1?

Reply:
I think we apply aria attribute to hide the background when the modal is opened. But it was always `h1`.

Reply:
yeah I'm not the a11y expert, I was just wondering
if it works, that's great